homeassistant/util/aiohttp.py

```diff
@@ -1,4 +1,6 @@
 """Utilities to help with aiohttp."""
+from __future__ import annotations
```

Comment:
We don't seem to be using this here. Maybe I'm missing something. I guess it doesn't hurt to add it for future convenience.

Reply:
No you are correct, I'll remove it.